plugins/flow-visibility/clickhouse-monitor/main.go

```diff
@@ -43,8 +35,8 @@ const (
 	skipRoundsNum = 3
 	// Connection to ClickHouse times out if it fails for 1 minute.
 	connTimeout = time.Minute
-	// Retry connection to ClickHouse every 5 seconds if it fails.
-	connRetryInterval = 5 * time.Second
+	// Retry connection to ClickHouse every 10 seconds if it fails.
+	connRetryInterval = 10 * time.Second
```

Comment:
Just curious why we changed this retry interval this time?

Reply:
I observed 4 to 5 retries at the beginning of the monitor as the ClickHouse server was not ready. Thought it would make sense to slightly lower the retry frequency. 
I've thought about adding some sleeping time before the monitor starts, but it seems that does not make sense to sleep first if the monitor is recovered from a the crash.